src/main/java/uk/gov/companieshouse/api/accounts/service/RptTransactionService.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.companieshouse.api.accounts.service;
+
+public interface RptTransactionService {
```

Comment:
Just noticed this, I believe we have a "MultipleResourceService<T>" which we should implement, it comes with a findAll() and deleteAll() method. So could do with deleting this service and then in the RptTransactionServiceImpl we can do "implements MultipleResourceService<RptTransaction>

Reply:
Already implement the MultipleResourceService so will just remove the interface 👍